packages/editor/src/hooks/pattern-partial-syncing.js

```diff
@@ -29,7 +29,7 @@ const {
  *
  * @return {Component} Wrapped component.
  */
-const withPartialSyncingControls = createHigherOrderComponent(
+const withPatternOverrideControls = createHigherOrderComponent(
```

Comment:
We should probably complete the job of renaming partial syncing here to pattern overrides by also updating the file name.